main.cpp

```diff
@@ -19,13 +19,30 @@ using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
 public:
-    CastCallBack(Rewriter& rewriter) {
-        // Your code goes here
-    };
-
+    CastCallBack(Rewriter& rewriter): _rewriter(rewriter) {};
+    
     void run(const MatchFinder::MatchResult &Result) override {
-        // Your code goes here
+        auto *cast = Result.Nodes.getNodeAs<CStyleCastExpr>("cast");
+        auto type = Lexer::getSourceText(

```

Comment:
Please describe what the `getSourceText()` function does.

Reply:
`getSourceText()` returns a string in the given range. 
In this code it is used to get string representation of cast type.